docs/sources/flow/reference/components/pyroscope.ebpf.md

```diff
@@ -0,0 +1,178 @@
+to
+the list of receivers passed in
+`forward_to`.
+
+Multiple `pyroscope.ebpf` components can be specified by giving them different labels, however it is not recommended.
+
+## Usage
+
+```river
+pyroscope.ebpf "LABEL" {
+  forward_to = RECEIVER_LIST
+}
+```
+
+## Arguments
+
+The component configures and starts a new ebpf profiling job to collect performance profiles from the current host.
+
+The following arguments can be used to configure a `pyroscope.ebpf`. Only the
+`forward_to` field is required and any omitted fields take their default
+values.
+
+| Name                      | Type                     | Description                                                            | Default | Required |
+|---------------------------|--------------------------|------------------------------------------------------------------------|---------|----------|
+| `forward_to`              | `list(ProfilesReceiver)` | List of receivers to send collected profiles to.                       |         | yes      |  
+| `targets`                 | `list(map(string))`      | List of targets to group profiles by container id                      |         | no       |   
+| `default_target`          | `map(string)`            | Default target to use when a PID is not associated with a container id |         | no       |    
+| `targets_only`            | `bool`                   | A flag to ignore profiles not associated with a container id           | true    | no       |      
+| `collect_interval`        | `duration`               | How frequently to collect profiles                                     | `10s`   | no       |       
+| `sample_rate`             | `int`                    | How many times per second to collect profile samples                   | 100     | no       |     
+| `pid_cache_size`          | `int`                    | The size of the pid -> proc symbol table LRU cache                     | 32      | no       |      
+| `elf_cache_size`          | `int`                    | The size of the elf file -> symbols LRU cache                          | 128     | no       |       
+| `container_id_cache_size` | `int`                    | The size of the pid -> container ID  LRU cache                         | 64      | no       |       
+| `collect_user_profile`    | `bool`                   | A flag to enable/disable collection of userspace profiles              | true    | no       |       
+| `collect_kernel_profile`  | `bool`                   | A flag to enable/disable collection of kernelspace profiles            | true    | no       |       
+
+## Exported fields
+
+`pyroscope.ebpf` does not export any fields that can be referenced by other
+components.
+
+## Component health
+
+`pyroscope.ebpf` is only reported as unhealthy if given an invalid
+configuration.
+
+## Debug information
+
+`pyroscope.ebpf` does not expose any component-specific debug information.
```

Comment:
Not for this PR but I think in the future we probably want to show active targets. May be we should create an issue for this.

Reply:
I think you added this ? We should update the doc then.